local-volume/provisioner/pkg/common/common.go

```diff
@@ -122,6 +144,18 @@ func CreateLocalPVSpec(config *LocalPVConfig) *v1.PersistentVolume {
 	}
 }
 
+// GetPathFromPV gets the local path (within provisioner container) of the PV
+func GetPathFromPV(pv *v1.PersistentVolume, config MountConfig) (string, error) {
+	specPath := pv.Spec.Local.Path
+	relativePath, err := filepath.Rel(config.HostDir, specPath)
+	if err != nil {
+		return "", fmt.Errorf("Could not get relative path for pv %q: %v", pv.Name, err)
+	}
+	mountPath := filepath.Join(config.MountDir, relativePath)
```

Comment:
I think we could do without specPath and mountPath varibles, and simply place the variable/function onto the single line referencing each of those variables.

Reply:
fixed